### ISSUE TITLE
Value Conversion Error

### ISSUE THREAD

**Issue by JorgeAraujo123** [webbankir/terraform-provider-apisix#2]
Hi and thanks for the Terraform provider for APISIX.

I'm having a problem using the provider. When installing a simple upstream:
```
resource "apisix_upstream" "test" {
  name = "test"
  type = "roundrobin"
}
```

I get the error:
```
│ Error: Value Conversion Error
│ 
│   with apisix_upstream.test,
│ An unexpected error was encountered trying to convert tftypes.Value into model.UpstreamType. This is always an error in the provider. Please report the following to the provider developer:
│ 
│ error retrieving field names from struct tags: need a struct tag for "tfsdk" on UpstreamWithoutIDType
```

I'm i doing something wrong?
Any help is appreciated.

**Comment by suslovsergey** [comment]
Hi! @JorgeAraujo123 please tell me what version of apisix do you used?

**Comment by JorgeAraujo123** [comment]
Hi @suslovsergey . The version is 2.14.1

**Comment by suslovsergey** [comment]
@JorgeAraujo123  i released  v0.0.26, check this version please.  please note that the upstream resource has not been tested in our environment (i mean apisix_upstream)

**Comment by JorgeAraujo123** [comment]
Yes it's working now, thank you @suslovsergey !
